chain/network/src/peer_manager/tests/routing.rs

```diff
@@ -28,9 +28,19 @@ use std::collections::HashSet;
 use std::net::Ipv4Addr;
 use std::sync::Arc;
 
+fn abort_on_panic() {
+    let orig_hook = std::panic::take_hook();
+    std::panic::set_hook(Box::new(move |panic_info| {
+        orig_hook(panic_info);
+        // TODO(gprusak): print stacktrace?
```

Comment:
Doesn’t the original hook do so when `RUST_BACKTRACE=1` is set?

Reply:
yes, it does. thx